src/config/mongoose_config_spec.erl

```diff
@@ -936,10 +938,9 @@ s2s_address() ->
        process = fun ?MODULE:process_s2s_address/1
       }.
 
-%% Callbacks for 'process'
```

Comment:
I think this comment remains valid, because it just says that the functions below are callbacks to the 'process' step.

Reply:
I have replaced it with the comment:
%% Callback for root level process function
for each callback function.
Is it ok?